packages/fxa-payments-server/src/routes/Subscriptions/SubscriptionItem.tsx

```diff
@@ -84,7 +87,7 @@ export const SubscriptionItem = ({
         </header>
 
         {!customerSubscription.cancel_at_period_end &&
-        subsequentInvoice?.total &&
+        (subsequentInvoice?.total || subsequentInvoice?.total === 0) &&
```

Comment:
Since the changes are in same scope, perhaps assign `(subsequentInvoice?.total || subsequentInvoice?.total === 0)` a name (naming is difficult I know).